src/components/opening-hours/opening-hours.vue

```diff
@@ -68,6 +76,49 @@ export default {
   },
   computed: {
     ...mapState(['isMobile']),
+    timeSlots() {
+      const { openingHoursSpace, openingHoursBuilding } = this
+      const newOpeningHours = openingHoursSpace.map((openingHour, index) => {
+        const buildingStart = openingHoursBuilding[index].time.length ? openingHoursBuilding[index].time[0][0] : []
+        const buildingEnd = openingHoursBuilding[index].time.length ? openingHoursBuilding[index].time[0][1] : []
+        const newTimeslots = []
+
+        newTimeslots.day = openingHour.day
+
+        newTimeslots.times = openingHour.time.reduce((timeslots, timeslot, index) => {
```

Comment:
Please be aware that within the scope of this reduce, the `index` variable is overwriting the `index` of the map on line 81. I think it would be good to call the `index` of the map something like `dayIndex` and the index here something like `slotIndex`.